exercises/grep/.gitignore

```diff
@@ -1,4 +1,9 @@
-# Ignore Cargo.lock if creating a library
+*.txt
+# Generated by Cargo
+# will have compiled files and executables
+/target/
+**/*.rs.bk
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
 # More information here http://doc.crates.io/guide.html#cargotoml-vs-cargolock
 Cargo.lock
-*.txt
```

Comment:
This line should not be removed; the grep exercise generates some text files, and while it is meant to clean up after itself, it might not in the event of panics.

Reply:
It's moved to the top line. Still there. I'll move it back to the bottom

Reply:
Using your explanation as a comment there too. Thanks!